Get-help.md

```diff
@@ -0,0 +1,139 @@
+Here we document Oppia's main communication channels and how to ask good questions.
+
+## Communication channels
+
+If you need help, there are a few communication channels you can use.  Developers usually respond within 24 hours, so long as you use a channel they actually check.
+
+### Email
+
+We have several mailing lists in the form of Google Groups that you can join:
+
+* [oppia-announce](https://groups.google.com/forum/#!forum/oppia-announce) is for announcements of new releases or blog posts. It's not for asking questions though.
+* [oppia-dev](https://groups.google.com/forum/#!forum/oppia-dev) is the main mailing list for communication between developers and for technical questions. You can post to it even if you're not a member of the group. This is where you can ask questions, solicit feedback, or make developer-specific announcements (e.g. a temporary GitHub outage).
+
+You can also email developers directly! Developers usually check their email pretty frequently, but figuring out their email addresses can be tricky. Some developers have their email addresses in their GitHub profile. If you know the developer's name, you can also look them up in the [`CONTRIBUTORS` file](https://github.com/oppia/oppia/blob/develop/CONTRIBUTORS). The addresses in the `CONTRIBUTORS` file aren't kept up to date though, so they might not work.
+
+You can ask your mentor to put you in touch with a developer if you don't know their address.
```

Comment:
Do we need to explain how to find a mentor? (I.e. fill out the contributor survey)

Reply:
done